src/main/java/org/codinjutsu/tools/jenkins/view/BrowserPanel.java

```diff
@@ -498,6 +498,11 @@ private void loadJobs() {
             } else {
                 jobList = requestManager.loadJenkinsView(currentSelectedView);
             }
+            if (jenkinsAppSettings.isAutoLoadBuilds()) {
+                for(Job job : jobList) {
+                    job.setLastBuilds(requestManager.loadBuilds(job));
+                }
+            }
```

Comment:
Should this setting load for all jobs recursively?
If not the setting should be named something like: _Auto load Build on first level_

Following Build is not loaded with this.
![grafik](https://user-images.githubusercontent.com/4604206/95239689-9b69d980-080b-11eb-8be3-bf2d516c4806.png)

Reply:
This setting load build on first level
I will change the description to:  Auto load Build on first level